### DISCUSSION TITLE
GUI fails because of CUDA

### DISCUSSION THREAD

**Discussion by KansaiUser** [colmap/colmap#2389] [Q&A]
I suppose this should be very simple but I would appreciate some help

I am running COLMAP from a container. When I try to do the Automatic Reconstruction the GUI closes with

`E0201 13:16:57.494634    29 cudacc.cc:59] CUDA error at /colmap/src/colmap/util/cuda.cc:57 - CUDA driver version is insufficient for CUDA runtime version`

What can I do to correct this?

**Comment by KansaiUser** [comment]
The previous situation was in my ubuntu 18 machine. I have also installed directly on my windows 10 machine (with GPU) and the same thing happened.

**Comment by YYSK-YMR** [comment]
I have the same problem on my windows11

**Comment by soryy708** [comment]
I tried updating to the latest NVIDIA drivers (with my _GeForce 2070 SUPER_ GPU) and the error changed from the above to:

```
cudacc.cc:59] CUDA error at /colmap/src/colmap/util/cuda.cc:56 - named symbol not found
```

**Reply by soryy708** [reply]
https://github.com/colmap/colmap/issues/4088
